www/activities/diary/js/diary.js

```diff
@@ -170,6 +170,33 @@ app.Diary = {
     el.innerText = dateString;
   },
 
+  sendStatistics: async function () {
+    if (app.Settings.get("statistics", "send-data") != true) {
+      return;
+    }
+
+    let entry = await this.getEntryFromDB(); // Get diary entry from DB
+
+    // Populate groups and get overal nutrition
+    if (entry) {
+      let totalNutrition = await app.FoodsMealsRecipes.getTotalNutrition(entry.items, "ignore");
+
+      let entryDetails = await Promise.all(entry.items.map(async (data) => {
+        return await app.FoodsMealsRecipes.getItem(data.id, data.type, data.portion, data.quantity);
+      }));
+
+      // Send nutrition and diary to target service
+      await app.Utils.timeoutFetch(app.Settings.get("statistics", "send-data-address"), {
+        headers: {
+          "User-Agent": "Waistline - Android - Version " + app.version + " - https://github.com/davidhealey/waistline",
+          "Authorization": app.Settings.get("statistics", "your_authorization_header")
```

Comment:
I believe you need to change `your_authorization_header` to `send-data-authorization` here.

Reply:
ohh. yes. done.